src/oas3/service.ts

```diff
@@ -35,11 +36,16 @@ export const transformOas3Service: Oas3HttpServiceTransformer = ({ document }) =
     map(document.servers, server => {
       if (!server) return null;
 
-      return {
+      const serv: IServer = {
         name: document.info?.title ?? '',
         description: server.description,
         url: server.url ?? '',
       };
+
+      const variables = translateServerVariables(server.variables);
+      if (Object.keys(variables).length) serv.variables = variables;
```

Comment:
Why are you checking the length? Can't we set the variables object anyway?

Reply:
Yep we can. I was trying to avoid setting a property that is undefined if we didn't need to, but if that's not an issue I can change it